### DISCUSSION TITLE
Prisma transaction delete with update

### DISCUSSION THREAD

**Discussion by vrg-success** [prisma/prisma#3954] [Q&A] [answered]
Hi,  do support prisma transaction delete with update?
I tried do that code, but transaction not working...

```ts
  const deleteResetPassRow = prisma.resetPasswords.delete({ where: { id: uuid } });
  const updateUserPass = prisma.users.update({
    where: { id: resetRow.userId },
    data: {
      password: 123, // Specify number to get error from prisma, expect that will rollback deleted row from resetPasswords table
    },
  });

  await prisma.$transaction([deleteResetPassRow, updateUserPass]);
```

**Comment by P4sca1** [comment]
I would also expect `deleteResetPassRow` to be reverted when `updateUserPass` errors. This seems to be a bug, so you might want to create a detailed bug report.

**Comment by ryands17** [comment]
Hey @vrg-success 👋 
Have you created an issue for the above?

**Reply by vrg-success** [reply, marked as answer]
Hi https://github.com/prisma/prisma/issues/3947

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
